modAionImpl/src/org/aion/zero/impl/cli/Cli.java

```diff
@@ -113,6 +113,64 @@ public int call(final String[] args, final Cfg cfg) {
+                            System.out.println("No "+keystoreDir.getPath()+" directory found. Creating directory ...");
+
+                            if (keystoreDir.mkdir()) {
+                                System.out.println("Created "+keystoreDir.getPath()+" directory.");
+                            } else {
+                                System.out.println("Directory: "+keystoreDir.getPath()+" could not be created. " +
+                                        "Please check user permissions or create directory manually.");
+                                System.exit(1);
+                            }
+                            System.out.println();
+                        }
+
+                        String certName = null;
+                        String certPass = null;
+                        Console console = System.console();
+                        // https://docs.oracle.com/javase/10/docs/api/java/io/Console.html
+                        // if the console does not exist, then either:
+                        // 1) jvm's underlying platform does not provide console
+                        // 2) process started in non-interactive mode (background scheduler, redirected output, etc.)
+                        // don't wan't to compromise security in these scenarios
+                        if (console == null) {
+                            System.out.println("No console found. This command can only be run interactively.");
+                            return 1;
+                        } else {
+                            console.printf("Enter name for certificate:\n");
+                            certName = console.readLine();
+                            certPass = String.valueOf(
+                                console.readPassword("Enter password for certificate (>6 characters):\n"));
+                        }
+
+                        if (certName == null || certPass == null) throw new IllegalStateException("Received " +
+                                "null values for ssl certificate name and password.");
+
+                        List<String> scriptArgs = new ArrayList<>();
+                        scriptArgs.add("/bin/bash");
+                        scriptArgs.add("script/create_cert.sh");
+                        scriptArgs.add(certName);
+                        scriptArgs.add(certPass);
+                        // add the hostname and ip optionally passed in as cli args
+                        scriptArgs.addAll(Arrays.asList(Arrays.copyOfRange(args, 2, args.length)));
+
+                        new ProcessBuilder(scriptArgs)
+                            .redirectInput(Redirect.INHERIT)
+                            .redirectOutput(Redirect.INHERIT)
+                            .redirectError(Redirect.INHERIT)
+                            .start()
+                            .waitFor();
+                    } else {
+                        printHelp();
```

Comment:
I think the global print help is sufficient for now. If you have any specific help string that you'd like to provide, I'll let @aionick make that change to the PR, since this was his feature :)

Reply:
I'll get to this tomorrow, if the global help isn't descriptive enough I'll add a specific one in for the ssl